docs/source/keywords.rst

```diff
@@ -131,44 +131,47 @@ General options
       * Build tests
       * ``OFF``
 
-    * * ``Kokkos_ENABLE_TUNING``
-      * Create bindings for tuning tools
-      * ``OFF``
-
-    * * ``Kokkos_ENABLE_DEBUG``
-      * Activate extra debug features - may increase compile times
-      * ``OFF``
-
-    * * ``Kokkos_ENABLE_DEBUG_BOUNDS_CHECK``
-      * Use bounds checking - will increase runtime
-      * ``OFF``
-
-    * * ``Kokkos_ENABLE_DEBUG_DUALVIEW_MODIFY_CHECK``
-      * Debug check on dual views
-      * ``OFF``
-
     * * ``Kokkos_ENABLE_DEPRECATED_CODE_3``
-      * Enable deprecated code in the Kokkos 3.x series
+      * Enable deprecated code in the Kokkos 3.x series :red:`[Removed in 4.3]`
       * ``OFF``
 
     * * ``Kokkos_ENABLE_DEPRECATED_CODE_4``
       * Enable deprecated code in the Kokkos 4.x series
       * ``ON``
 
+    * * ``Kokkos_ENABLE_DEPRECATION_WARNINGS``
+      * Whether to raise warnings at compile time when using deprecated Kokkos facilities
+      * ``ON``
+
+    * * ``Kokkos_ENABLE_TUNING``
+      * Create bindings for tuning tools
+      * ``OFF``
+
     * * ``Kokkos_ENABLE_AGGRESSIVE_VECTORIZATION``
       * Aggressively vectorize loops
       * ``OFF``
 
-    * * ``Kokkos_ENABLE_COMPILER_WARNINGS``
-      * Print all compiler warnings
+Debugging
+---------
+.. list-table::
+    :widths: 25 65 35
+    :header-rows: 1
+    :align: left
+
+    * -
+      - Description/info
+      - Default
+
+    * * ``Kokkos_ENABLE_DEBUG``
+      * Activate extra debug features - may increase compile times
```

Comment:
Should this be elaborated a little bit more? This flag sounds very general, but it's also impossible to tell what the "extra debug features" would be without looking into the code base.

Reply:
I have not modified the description.  Feel free to push if you want to update as part of this PR.